shared/middleware/helpers/redis_table.rb

```diff
@@ -77,30 +72,26 @@ def fetch(id)
     make_row(id, hash)
   end
 
-  # Updates an existing row, notifying other clients using the pubsub service.
+  # Updates an existing row.
   #
   # @param [Integer] id The id of the row to update.
   # @param [Hash] hash The updated hash.
   # @param [String] ignored_ip Unused, for compatability with other table apis.
   def update(id, hash, ignored_ip=nil)
     @props.set(row_key(id), hash.to_json)
-    publish_change({:action => 'update', :id => id})
     merge_id(hash, id)
   end
 
-  # Deletes a row, notifying other clients using the pubsub service.
+  # Deletes a row.
   #
   # @param [Integer] id The id for the new row
   def delete(id)
-    deleted = @props.delete(row_key(id))
-    publish_change({:action => 'delete', :id => id}) if deleted
-    deleted
+    @props.delete(row_key(id))
   end
 
   # Deletes all the tables and rows in a shard.
-  def self.reset_shard(shard_id, redis, pub_sub)
+  def self.reset_shard(shard_id, redis)
     RedisPropertyBag.new(redis, shard_id).delete_all
-    pub_sub.publish(shard_id, 'all_tables', {:action => 'reset_shard'}) if pub_sub
```

Comment:
Since pubsub is now a concern of the netsim api layer, I was assuming that layer would do the publish in its reset_shard operation once it has one.